frontend/packages/data-portal/app/utils/featureFlags.ts

```diff
@@ -0,0 +1,45 @@
+import { useSearchParams } from '@remix-run/react'
+
+import { useEnvironment } from 'app/context/Environment.context'
+
+export type FeatureFlagEnvironment = typeof process.env.ENV
+
+export const FEATURE_FLAGS = {
+  methodType: 'dev',
```

Comment:
We should update the data structure so that we can enable a feature flag for multiple environments.